_pages/en_US/troubleshooting.txt

```diff
@@ -155,3 +155,16 @@ Browser based exploits (such as browserhax or 2xrsa) are often unstable and cras
 1. Your device is bricked
 1. You will need to get an ntrboot-comptible flashcart (one of the ones on [this list](ntrboot) or a [hardmod](https://gbatemp.net/threads/414498/), or repair / replace your device
 1. For support (in English), join [Nintendo Homebrew on Discord](https://discord.gg/MWxPgEp)
+
+## System update error after installing CFW
+
+  + You get an error while trying to use WiFi (during system update)
```

Comment:
An error when trying to use WiFi during system update, as opposed to trying to use something else?...

The best way to clarify this section would be to show that the error that people should see is "An error has occurred".